java/postgresql-jdbc/Portfile

```diff
@@ -23,25 +23,26 @@ distname            ${name}-${version}.src
 worksrcdir          ${name}-${version_major}.${version_minor}-${version_patch}.src
 
 master_sites        ${homepage}/download/
-checksums           rmd160  215bcc04ce3aa41266fc1fcb27a31a95a710c7bc \
-                    sha256  97ee866689f0e6c45f3e71bcb88f42f16ea9a8a79965e6790efb15e133585029 \
-                    size    1404018
+checksums           rmd160  017982ae8dfd0beb6e78befd45ee1181fbc84c97 \
+                    sha256  65b2385b7b627bb7369ce9d22d1d92a1c928a74eff93a120504c236f7685883c \
+                    size    1567710
 
 java.version        1.8*
 java.fallback       openjdk8
-depends_build       port:maven32
+depends_build       port:gradle
 
 use_configure       no
 
-build.cmd           mvn32
-build.target        package
-build.args          --define skipTests \
-                    --define maven.repo.local=${workpath}/m2/repository
+build.cmd           gradle
```

Comment:
This should use an absolute path to the Macports gradle executable. Here's my untested change:

`build.cmd           ${prefix}/bin/gradle
`